src/app/delegates/page.tsx

```diff
@@ -1,50 +1,44 @@
-import { Markdown } from '@/components/Markdown';
-import { Card, CardContent } from '@/components/ui/card';
-import { Copy, Hexagon } from 'lucide-react';
+'use client';
+
+import DelegateCard from '@/components/common/DelegateCard';
+import { Button } from '@/components/ui/button';
+import { ScrollArea } from '@/components/ui/scroll-area';
+import { DelegatesProvider, useDelegates } from '@/context/DelegatesContext';
+
+const DelegatesList = () => {
+  const { displayedDelegates, loadMore, delegates } = useDelegates();
+
+  return (
+    <ScrollArea className="mt-4">
+      <div className="grid grid-cols-1 md:grid-cols-2 gap-7">
+        {displayedDelegates.map((delegate) => (
+          <DelegateCard key={delegate.address} delegate={delegate} />
+        ))}
+      </div>
+      {displayedDelegates.length < delegates.length && (
+        <div className="mt-4 flex justify-center">
+          <Button onClick={loadMore} variant="outline">
+            Load More
+          </Button>
+        </div>
+      )}
+    </ScrollArea>
+  );
+};
 
 const Delegates = () => {
   return (
-    <>
+    <DelegatesProvider>
```

Comment:
no need to wrap this with DelegatesProvider if you have already wrapped the context/index.tsx